### PR TITLE
Updated the repo links and renamed the artifact produced

### PR DESCRIPTION
**Pull request by ribafish**
Updated the repo links and renamed the artifact produced as per the email thread.

@runningcode I'm not sure where to apply the Maven name discussed in the email.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -30,7 +30,7 @@ lazy val sbtCommonCustomUserDataPlugin = (project in file("."))
       "-Xlint:-unused",
       "-Ywarn-unused:imports,privates,locals,implicits"
     ),
-    name := "sbt-common-custom-user-data",
+    name := "sbt-develocity-common-custom-user-data",
```
Comment (runningcode) [inline build.sbt:33]: Is this the maven name or group-id?

Comment (ribafish) [reply build.sbt:33]: That's the artifact name

**Comment by runningcode** [comment]
Can you please look in to this?

> I'm not sure where to apply the Maven name discussed in the email.

**Comment by ribafish** [comment]
> Can you please look in to this?
> 
> > I'm not sure where to apply the Maven name discussed in the email.

@runningcode I did and I can't find a way to set it the way a `pom.xml` allows - I'm not even sure where the discussion about this came from in the email, as I can't find a mention about a maven name in the [slack thread](https://gradle.slack.com/archives/C04PQJ0F4GG/p1710856496393689?thread_ts=1710851329.473389&cid=C04PQJ0F4GG) where the discussion started in the first place. Maybe @gradle/ge-sbt has some ideas?

**Review by Duhemm — APPROVED** [review]
Code looks good to me, but I don't know what's been decided regarding the name of the plugin. Thanks!

**Review by runningcode — APPROVED** [review]
Names look correct to me 👍

**Pull request merged**